src/Tickets/Seating/app/admin/layoutEdit/index.js

```diff
@@ -37,6 +39,10 @@ export async function init(dom) {
 		handleReservationsUpdatedFollowingSeatTypes
 	);
 
+	registerAction(GO_TO_ASSOCIATED_EVENTS, ( data ) => {
+		window.location.href = getAssociatedEventsUrl(data.layoutId);
```

Comment:
Check if `data.layoutId` is defined before firing the action.

Reply:
Add a test for this in the `tests/slr_jest/admin/layoutEdit.spec.js` file:

- check the action is registered
- check firing the action without a `layoutId` will not redirect the user
- check firing the action with a `layoutId` will redirect the user